test-avocado/compiletest.sh

```diff
@@ -16,7 +19,7 @@ test -d $PACKAGE/$BUILD && /bin/rm -r $PACKAGE/$BUILD
 cd $PACKAGE
 mkdir -p $BUILD
 cd $BUILD
-../autogen.sh --prefix=/usr --enable-maintainer-mode --enable-debug
+../autogen.sh --prefix=/usr --libdir=/usr/lib$ARCH64 --enable-maintainer-mode --enable-debug
```

Comment:
Yes, for now I'll maintan that, then it is possible that it will be not part of base CI testing, and in CI will be used tools/make-* and this test will be only recompilation test, not installing cockpit via make install